packages/date-picker/src/DatePicker.tsx

```diff
@@ -19,38 +19,34 @@ export const DatePickerWrapper: React.FunctionComponent<DatePickerProps> = ({
   onDayChange,
   labelText,
   isDisabled = false,
-}) => (
-  <>
-    {labelText && <Label disabled={isDisabled} labelText={labelText} />}
-    <div
-      className={classnames(
-        styles.wrapper,
-        styles.withStartIconAdornment,
-        styles.withSearch,
-        {
-          [styles.withDisabled]: isDisabled,
+}) => {
+  const getNavbar = ({ ...navbarProps }: CalendarNavProps) => (
+    <CalendarNav {...navbarProps} />
+  )
+
+  return (
+    <>
+      <TextField
+        id="date-picker"
+        labelText={labelText}
+        icon={dateStart}
+        inputValue={
+          selectedDate ? selectedDate.toLocaleDateString() : undefined
         }
-      )}
-    >
-      <div className={styles.startIconAdornment}>
-        <Icon icon={dateStart} role="presentation" />
-      </div>
-      <DayPickerInput
-        value={selectedDate}
-        onDayChange={onDayChange}
-        dayPickerProps={{
-          selectedDays: selectedDate,
-          disabledDays: {
-            daysOfWeek: [0, 6],
-          },
-        }}
-        inputProps={{
-          disabled: isDisabled,
-          className: classnames(styles.input, {
-            [styles.disabled]: isDisabled,
-          }),
+        disabled={isDisabled}
+      />
+
+      <DayPicker
+        selectedDays={selectedDate}
+        // TODO create disabledDays prop, this is to display disabled styling
+        disabledDays={{
+          daysOfWeek: [6],
         }}
+        onDayClick={onDayChange}
+        navbarElement={getNavbar}
+        classNames={defaultDatePickerClasses}
+        className={styles.calendar}
```

Comment:
/nit `classNames` and `className`? Could these be combined or otherwise simplified?

Reply:
From our initial look, it's separated and we couldn't combine them but I'll take another look.